modules/exploits/windows/smb/ms17_010_eternalblue.rb

```diff
@@ -5,14 +5,17 @@
 
 require 'ruby_smb'
 require 'ruby_smb/smb1/packet'
+require 'rubyntlm'
+require 'net/ntlm'
 require 'windows_error'
 
 class MetasploitModule < Msf::Exploit::Remote
   Rank = AverageRanking
 
-  include Msf::Exploit::Remote::DCERPC
```

Comment:
Looks like we'll need the `rhost` method available for the `check` method, which for some reason comes from the tcp mixin 🤷 

```ruby
  include Msf::Exploit::Remote::Tcp
```